_docs-sources/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual-steps.md

```diff
@@ -0,0 +1,137 @@
+root user. Follow the manual steps outlined in the [AWS docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#id_root-user_manage_mfa) for configuring a MFA device.
+
+- (Recommended) Configure a hardware MFA device. We suggest using a
+[Yubikey](https://www.yubico.com/) due to its strong security characteristics and multitude of form
+factors. Refer to
+[the AWS documentation for more information on using a hardware device with the root user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_physical.html#enable-hw-mfa-for-root). If you're targeting the CIS AWS Foundations Benchmark, this is part of recommendation 1.6.
+
+- Alternatively, if you don't have access to a hardware MFA device, configure a virtual MFA device. This is still vastly more secure than no MFA device at all!
+
+## 3. Answer security questions and complete contact details
+
+When setting up a new account, AWS asks for contact information and security questions. Unfortunately, there is no API or automation available for this functionality. In the AWS console, visit the [Account settings](https://console.aws.amazon.com/billing/home?#/account) page and complete the _Alternate Contacts_ and _Configure Security Challenge Questions_ questions.
+
+
+## 4. Delete default VPCs and rules from default security groups
+
+Use [cloud-nuke](https://github.com/gruntwork-io/cloud-nuke) to remove the rules from the default VPC and the default ingress/egress rules from the default security groups. Note that it isn’t possible to actually delete the default security group, so instead the command deletes the rules, eliminating the risk of something being mistakenly exposed.
+
+Authenticate to each account (shared, security, dev, prod etc), and then run the following command:
+
+Usage:
+```bash
+cloud-nuke defaults-aws
+```
+
+Example:
+```bash
+aws-vault exec dev -- cloud-nuke defaults-aws
+```
+
+## 5. Subscribe to `BenchmarkAlarmTopic` Alarms
+
+The AWS Config alerts and CloudWatch Metric Alarms all go to the SNS topic `BenchmarkAlarmTopic`. Unfortunately, there is no way to automate
+subscribing to the SNS topic as each of the steps require validating the delivery target. For each deployed account, follow the steps outlined in
+the [AWS docs](https://docs.aws.amazon.com/sns/latest/dg/sns-user-notifications.html) to be notified by Email, Phone,
+or SMS for each of the alerts.
+
+You can also configure an automated system integration if you have a third party alerting system or central dashboard.
+Follow the steps in the [AWS
+docs](https://docs.aws.amazon.com/sns/latest/dg/sns-http-https-endpoint-as-subscriber.html) on how to add an HTTPS endpoint as a subscriber to the alerts.
+
+## 6. Manually maintain buckets to analyze in the `buckets_to_analyze` variable
+
+To set up Macie to analyze the desired S3 buckets, you’ll need to create a **Macie classification job**. Typically,
```

Comment:
What is Macie and why do I care? How does it relate to the CIS-Benchmark? If I 'm scanning this section as a new user who has never heard the word Macie before - what would I want to know in order to make forward progress? 

We should probably include a simple sentence in the beginning explaining that Macie is an AWS service that can be configured to scan S3 buckets for Personally Identifiable Information (PII) and how this relates to the CIS-Benchmark.

Reply:
I got an definition from AWS -- it says that Macie "discovers and protects sensitive data" WDYT?